dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -1715,7 +1793,29 @@ void ScrollViewer::HandleKeyDownForXYNavigation(winrt::KeyRoutedEventArgs args)
 
         if (shouldMoveFocus)
         {
-            winrt::FocusManager::TryFocusAsync(nextElement, winrt::FocusState::Keyboard);
+            auto focusAsyncOperation = winrt::FocusManager::TryFocusAsync(nextElement, winrt::FocusState::Keyboard);
+
+            if (SharedHelpers::IsAnimationsEnabled()) // When system animations are turned off, the bring-into-view operations are not turned into animations.
+            {
+                auto targetElement = nextElement.try_as<winrt::UIElement>();
```

Comment:
>auto targetElement = nextElement.try_as<winrt::UIElement>(); [](start = 16, length = 60)

I would move this into the lambda declaration: [..., targetElement = nextElement.try_as<winrt::UIElement>()]